### PR TITLE
flat dist dir

### PR DESCRIPTION
**Pull request by giovannicandido**
Part of pull request #22

### CODE REVIEW

**Comment by mattlewis92** [comment]
Just realised https://github.com/mattlewis92/generator-angular-library/pull/27 makes more sense to be part of this PR as they both alter the release script, can you cherry-pick it into this branch please? 😄

**Comment by mattlewis92** [comment]
Oh and this one as well actually: https://github.com/mattlewis92/generator-angular-library/pull/23/files

Just keep the config one separate, sorry to keep asking for changes!

File: generators/app/templates/_package.json [hunk 1]
```diff
@@ -18,8 +18,11 @@
     "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s",
     "typedoc": "typedoc --options typedoc.json src/*.ts",
     "gh-pages": "git checkout gh-pages && git merge master --no-edit && npm run build:demo && npm run typedoc && git add . && git commit -m 'chore: build demo and docs' && git push && git checkout master",
+    "copyfiles": "cp package.json LICENSE README.md dist/",
```
Comment (mattlewis92) [inline generators/app/templates/_package.json]: Can you add the CHANGELOG.md as well

**Comment by giovannicandido** [comment]
The check picks are starting to get conflicts and messing up with the linear history.

**Comment by mattlewis92** [comment]
@giovannicandido I'll just squash all the commits together when I merge this

**Comment by mattlewis92** [comment]
I just want to check this out locally and double check everything works OK before merging, but everything looks OK by eye 😄

**Comment by giovannicandido** [comment]
😄 Is a good idea. I use `npm link` to test locally

**Comment by mattlewis92** [comment]
@giovannicandido so sorry about the delay on reviewing this, I had surgery this week and have been out of action. Thanks a tonne for your patience and contribution 👍

**Pull request merged**